main.go

```diff
@@ -49,6 +56,28 @@ func startRun(config *common.CatalogConfig, rh request.Handler) {
 	rh.StartHandlingRequests(config, &towerapiworker.DefaultAPIWorker{})
 }
 
+func getConfigFile() (string, error) {
+	if fileExists("./rhc/workers/catalog.toml") {
+		return "./rhc/workers/catalog.toml", nil
+	}
+	usr, err := user.Current()
+	if err == nil && fileExists(fmt.Sprintf("%s/.config/rhc/workers/catalog.toml", usr.HomeDir)) {
+		return fmt.Sprintf("%s/.config/rhc/workers/catalog.toml", usr.HomeDir), nil
+	}
+	if fileExists("/etc/rhc/workers/catalog.toml") {
```

Comment:
@bzwei Should we check if we have the ability to read the file? The file might exist but we won't have access to read it.

Reply:
if the cannot be read, it will blow up when viper read it a few lines below.